CoreWidgetProvider/Widgets/Templates/SSHWalletPatternMatching.json

```diff
@@ -0,0 +1,57 @@
+{
+  "type": "AdaptiveCard",
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.5",
+  "body": [
+    {
+      "type": "Container",
+      "items": [
+        {
+          "type": "TextBlock",
+          "text": "%SSH_Widget_Template/PatternMatching%",
+          "wrap": true,
+          "horizontalAlignment": "Left",
+          "size": "Medium",
+          "weight": "Bolder"
```

Comment:
I'd love for Design to give this a quick peek before checking in if they haven't yet.

Reply:
Will do.